examples/generative/stylegan.py

```diff
@@ -73,7 +95,8 @@ def resize_image(res, sample):
 
 def create_dataloader(res):
     batch_size = batch_sizes[log2(res)]
-    dl = ds_train.map(partial(resize_image, res), num_parallel_calls=tf.data.AUTOTUNE)
+    # NOTE: we added the `unbatch` call in order to support dynamic batch sizes from KPL dataset
```

Comment:
It seems the `unbatch` call is because we need to use the `drop_remainder=True` option, because the model does not support dynamic batch sizes, right?

Reply:
Is dynamic batch size a new feature? At the time of writing the code with TF2.2 , it would crash unless `drop_remainder=True`.

Reply:
The "dynamic batch sizes" that I'm referring to are not a native Keras feature but rather trying to support the `batch_sizes` which scale with the resolution, [here](https://github.com/yarri-oss/keras-io/blob/patch-1/examples/generative/stylegan.py#L63)

And yes, we need to use `drop_remainde=True` which seems to then force us to use `unbatch` [here](https://github.com/yarri-oss/keras-io/blob/patch-1/examples/generative/stylegan.py#L99). I'm sure there's a more elegant way to handle this.

Reply:
Please update the comment to clarify this, e.g.
```
# NOTE: we unbatch the dataset so we can `batch()` it again with the `drop_remainder=True` option
# since the model only supports a single batch size
```

Reply:
Done